feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -313,15 +315,16 @@ func validateTrafficFlows(t *testing.T, ate *ondatra.ATEDevice, ateTop gosnappi.
 	ate.OTG().StopTraffic(t)
 	otgutils.LogFlowMetrics(t, ate.OTG(), ateTop)
 	otgutils.LogPortMetrics(t, ate.OTG(), ateTop)
-	flowPath := gnmi.OTG().Flow(flow)
-	got := gnmi.Get(t, ate.OTG(), flowPath.LossPct().State())
+	recvMetric := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(flow).State())
+	lostPackets := recvMetric.GetCounters().GetOutPkts() - recvMetric.GetCounters().GetInPkts()
+	got := lostPackets * 100 / recvMetric.GetCounters().GetOutPkts()
 	if drop {
 		if got != 100 {
-			t.Fatalf("Traffic passing for flow %s got %f, want 100 percent loss", flow, got)
+			t.Fatalf("Traffic passing for flow %s got %v, want 100 percent loss", flow, got)
```

Comment:
What's the reason for changing from a %f?

Reply:
Done..."got" is not floating point